magento/pwa-studio
# PWA-1550: PB verify video

## Description

To verify that the Page Builder video content type renders correctly on Venia for each content type property.
Only alignment is tested via Cypress automation since functionality wise it uses cross origin iframe videos which is not feasible for automation (nor needed really so if we were to test it it should be done manually).

## Related Issue





Closes [PWA-1550](https://jira.corp.magento.com/browse/PWA-1550): Verify Video

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps


1. Cypress `video.spec.js` should pass
Steps - 
Go to venia-integration-test dir
Run `yarn install`
Run `yarn test`
Run `video.spec.js` from cypress ui

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
